### DISCUSSION TITLE
Version 3.0 beta 5

### DISCUSSION THREAD

**Discussion by wrongkindofdoctor** [NOAA-GFDL/MDTF-diagnostics#402] [General]
New PODS:
* albedofb
* seaice_suite
* blocking_neale
* top heaviness metric
* TC_Rain
* Precipitation Buoyancy Diagnostic

Bug fixes:
* Fix issues with valid_range and byte type attributes
* Fix issue with empty var.associate_files in data_manager module

<hr /><em>This discussion was created from the release <a href='https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0.beta.5'>Version 3.0 beta 5</a>.</em>

**Comment by bitterbark** [comment]
obs_data files for the new pods will be coming soon to the NCAR ftp site.

**Reply by wrongkindofdoctor** [reply]
@bitterbark Excellent! Thank you.

**Reply by bitterbark** [reply]
Updated obs data is now posted under the ftp://ftp/archive/mdtf/obs_data_latest directory (symbolically linked to MDTF_v3.0-beta.5_obs_data/).

Note that security on the ftp site might prevent your browser from retrieving the data. More consistently successful methods include:
1)  wget ftp://ftp.cgd.ucar.edu/archive/mdtf/obs_data_latest/\*
2) search for the Globus collection "NCAR CGD Anon"

**Reply by wrongkindofdoctor** [reply]
@bitterbark Thanks! I have updated the README file to direct users to run wget on the link to the latest dataset, or download the data using Globus as described.

**Reply by bitterbark** [reply]
Whoops, I just realized that there is no data in the TC_Rain directory (including in the tar file)

**Reply by wrongkindofdoctor** [reply]
@bitterbark That's fine. The TC_Rain POD doesn't require input obs data, but there does need to be an empty dummy directory for the POD for functionality.
